src/app/hooks/useTOC.tsx

```diff
@@ -1,28 +1,14 @@
-import { useEffect, useState } from 'react';
+import { useEffect } from 'react';
 import { useDispatch } from 'react-redux';
-import { useLocation } from 'react-router-dom';
 import { TOGGLE_TOC } from '../redux/actions';
 
 export const useTOC = (toc: boolean): void => {
     const dispatch = useDispatch();
-    const { pathname } = useLocation();
-    const [prevTOCPathname, setPrevTOCPathname] = useState('');
```

Comment:
For future reference, we have a hook called 'usePrevious' that you can use to check a value against its previous value, if you run into a situation where you need something like that. I've used it in cases where I needed to compare a value in the redux store against a previous value (like going from loading = true to loading = false, or things like that).